src/item/component/IconComponent.php

```diff
@@ -17,6 +17,7 @@ public function getName(): string {
 
 	public function getValue(): array {
 		return [
+			"texture" => $this->texture,
```

Comment:
this doesn't match the docs https://learn.microsoft.com/en-us/minecraft/creator/reference/content/itemreference/examples/itemcomponents/minecraft_icon?view=minecraft-bedrock-stable